src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,18 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def cve?
+    kind == "cve"
+  end
+
+  def verify_issue_name!(name)
```

Comment:
You didn't say something about putting this in the Issue model and passing the tracker in the params?